docs/source/integrations/ios-fairplay-sdk.md

```diff
@@ -1,138 +1,140 @@
- 
-Legacy method 1: Pass in a reference to your AVPlayer instance, the token to be used, and the asset ID when retrieving a license. 
- 
+
+Legacy method 1: Pass in a reference to your AVPlayer instance, the token to be used, and the asset ID when retrieving a license.
+
 let player = AVPlayer(url: assetURL!)
 self.drm = vudrmFairPlay(player: player, token: token, assetName: assetName)
-            
-Legacy method 2: Pass in a reference to your stream URL and the token to be used when retrieving a license. An AVAsset will be created by VUDRMFairPlay and this can be used to create your player. 
- 
+  
+Legacy method 2: Pass in a reference to your stream URL and the token to be used when retrieving a license. An AVAsset will be created by VUDRMFairPlay and this can be used to create your player.
+
 self.drm = vudrmFairPlay(url:assetURL! as NSURL, token: token, assetName: assetName)
 let playerItem = AVPlayerItem(asset: (drm?.asset)!)
-let player = AVPlayer(playerItem: playerItem) 
-3. Finally, present your player: 
-self.player = player let playerController = AVPlayerViewController() playerController.delegate = self as? AVPlayerViewControllerDelegate self.present(playerController, animated: true, completion: nil) playerController.view.frame = self.view.frame playerController.player = self.player player.play() 
-DEMO 
-With the VUDRMFairPlay framework installed, the example demo application can be initialised either via source code in the IDE, or using field entry on device.
-● To initialise via code in IDE, before running, simply replace the empty ViewController.swift values with your own valid values.
-private var assetURL = URL(string: "")
-    		private var token = ""
-   		private var assetName = ""
- 
-● To initialise via view controller on device, populate the URL, Token, and Asset Name fields with valid values. For ease of use, pasting into the URL field in the form URL + space + Token will auto populate both fields correctly. At runtime, any values in the URL and Token fields will override any hard coded values. 
-
-### Limitations ###
+let player = AVPlayer(playerItem: playerItem)
+
+3. Finally, present your player:
+   self.player = player let playerController = AVPlayerViewController() playerController.delegate = self as? AVPlayerViewControllerDelegate self.present(playerController, animated: true, completion: nil) playerController.view.frame = self.view.frame playerController.player = self.player player.play()
+   DEMO
+   With the VUDRMFairPlay framework installed, the example demo application can be initialised either via source code in the IDE, or using field entry on device.
+   ● To initialise via code in IDE, before running, simply replace the empty ViewController.swift values with your own valid values.
+   private var assetURL = URL(string: "")
```

Comment:
Surround code snippet in triple back-ticks

Reply:
any changes to this document will have to be done by Adam